kyo-kernel/shared/src/main/scala/kyo/Kyo.scala

```diff
@@ -504,8 +752,41 @@ object Kyo:
             else v.map(t => Loop.continue(acc.append(t)))
         }
 
-    private def toIndexed[A](source: IterableOnce[A]): Seq[A] =
+    private[kyo] def toIndexed[A](source: IterableOnce[A]): Seq[A] =
```

Comment:
Does this need to be package private now?
```suggestion
    private def toIndexed[A](source: IterableOnce[A]): Seq[A] =
```